ibm/service/scc/resource_ibm_scc_account_settings.go

```diff
@@ -19,100 +20,267 @@ import (
 
 func ResourceIBMSccAccountSettings() *schema.Resource {
 	return &schema.Resource{
-		CreateContext: resourceIbmSccAccountSettingsUpdate,
+		CreateContext: resourceIbmSccAccountSettingsCreate,
 		ReadContext:   resourceIbmSccAccountSettingsRead,
 		UpdateContext: resourceIbmSccAccountSettingsUpdate,
 		DeleteContext: resourceIbmSccAccountSettingsDelete,
 		Importer:      &schema.ResourceImporter{},
 
 		Schema: map[string]*schema.Schema{
-			"location_id": {
```

Comment:
You cann't remove a argument directly ..It breaks the backward compatability
You need to follow guidelines of Terraform removal/deprecation
https://www.terraform.io/plugin/sdkv2/best-practices/deprecations
New argument which is a required "loction" cann't be added which breaks all existig users

Reply:
In that case, it's best that i create the deprecation notice for it.